src/curated-corpus/components/ScheduleItemFormConnector/ScheduleItemFormConnector.tsx

```diff
@@ -102,10 +102,7 @@ export const ScheduleItemFormConnector: React.FC<
     });
 
   // What to do when the user clicks on a date in the calendar.
-  const handleDateChange = (
-    date: MaterialUiPickersDate,
-    value?: string | null | undefined
-  ) => {
+  const handleDateChange = (date: any, value?: string | null | undefined) => {
```

Comment:
I don't like setting the `date` var to `any` here and in the other part of the code that's moved to the new date pickers but the date picker upgrade has been troublesome enough! 🙃